### DISCUSSION TITLE
How to control swf game

### DISCUSSION THREAD

**Discussion by SivaG4K** [ruffle-rs/ruffle#10717] [General]
window.addEventListener("load", (event) => {
    const ruffle = window.RufflePlayer.newest();
    const player = ruffle.createPlayer();
    const container = document.getElementById("container");
    container.appendChild(player);

    player.load("Game1.swf");

    setTimeout(() => {
        player.gotoAndStop(10);
    }, 10000); // delay execution by 2 seconds
	});

This is my code but getting error"Uncaught TypeError: player.gotoAndStop is not a function".
How to control goto some frame and label outside of the swf?

**Comment by lvyitian** [comment]
There's currently no public API available to do so, you can add it by yourself or have a look at the ContextMenu(pop up when right clicked).
